frontend/rpm/template.go

```diff
@@ -109,23 +113,36 @@ func (w *specWrapper) Requires() fmt.Stringer {
 	runtimeKeys := dalec.SortMapKeys(deps.Runtime)
 	for _, name := range runtimeKeys {
 		constraints := deps.Build[name]
-		// satisifes is only for build deps, not runtime deps
+		// satisfies is only for build deps, not runtime deps
 		// TODO: consider if it makes sense to support sources satisfying runtime deps
 		writeDep(b, "Requires", name, constraints)
 	}
 
 	return b
 }
 
-func writeDep(b *strings.Builder, kind, name string, constraints []string) {
-	if len(constraints) == 0 {
-		fmt.Fprintf(b, "%s: %s\n", kind, name)
+func writeDep(b *strings.Builder, kind, name string, constraints dalec.PackageConstraints) {
+	do := func() {
+		if len(constraints.Version) == 0 {
+			fmt.Fprintf(b, "%s: %s\n", kind, name)
+			return
+		}
+
+		sort.Strings(constraints.Version)
```

Comment:
I believe this will mutate the underlying `constraints.Version` array, is that ok?

Reply:
In the end I'm not sure why I was sorting this.
I just removed it.